### PR TITLE
Small text changes

### PR DESCRIPTION
**Pull request by bradr5**
## Description
Overview Page
before: `Red Hat Advanced Cluster Security for Kubernetes RHACS Cloud Service lowers operational costs...`
after: `RHACS Cloud Service lowers operational costs...`

Also includes text changes to the Getting Started page

## Checklist (Definition of Done)

- [ ] Unit and integration tests added
- [ ] Added test description under `Test manual`
- [ ] Documentation added if necessary
- [ ] CI and all relevant tests are passing
- [ ] Add the ticket number to the PR title if available, i.e. `ROX-12345: ...`
- [ ] Discussed security and business related topics privately. Will move any security and business related topics that arise to private communication channel.

### CODE REVIEW

File: src/routes/GettingStartedPage/Wizard/InstallOptions.js
Comment (sachaudh) [inline src/routes/GettingStartedPage/Wizard/InstallOptions.js:72]: Should this be `Secured Cluster` or is it actually `SecuredCluster`?

**Review by sachaudh — APPROVED** [review]
Looks good to me

File: src/routes/GettingStartedPage/Wizard/InstallOptions.js
Comment (bradr5) [reply src/routes/GettingStartedPage/Wizard/InstallOptions.js:72]: I'm pretty sure it's supposed to be one word but I asked the same question. I'll wait on a response before I merge.

**Pull request merged**
